Source/SwiftLintFramework/Helpers/RegexHelpers.swift

```diff
@@ -8,7 +8,7 @@
 
 struct RegexHelpers {
     // A single variable
-    static let varName = "[a-zA-Z_][a-zA-Z0-9_]+"
+    static let varName = "[a-zA-Z_\.][a-zA-Z0-9_\.]+"
```

Comment:
There could be an enum or static var on a number.

Not saying this code make sense, but it compiles:
```extension Int {
    static var header = 50
}
NSMakeRange(.header, 0)
```